migrations/20210519172140_v1.11.0.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const sql = fs.readFileSync(__filename + '.sql', 'utf8');
+
+exports.up = async (db) => {
+	await db.schema.alterTable('package_version', (table) => {
+		table.dropForeign([ 'packageId' ]);
+		table.dropUnique([ 'packageId', 'version' ]);
+
+		table.string('type', 16).notNullable().defaultTo('version');
```

Comment:
Also looking at the changes, I'm thinking it might be a good idea to call this `versionType` to avoid confusion since everywhere else, `type` refers to package.

Reply:
> This could probably use an actual enum type?

I guess enum can be used for this column as the values are unlikely to change

> I'm thinking it might be a good idea to call this versionType to avoid confusion

I'm really not sure about that. At the DB level, it's fine to name it `type`. We shouldn't maintain unique column names across multiple tables. Better to use aliases if needed.

And from a code perspective, correct queries and proper variable names eliminates the problem.